### PR TITLE
[VUMM-748] h2: extract ddl modeldb schema

### PR DESCRIPTION
**Pull request by anandjakhaniya**
## Testing
- [x] Unit test
- [ ] Deployed to dev env
- [ ] Other (explain) 

## Reverting
- [ ] Contains Migration - _Do Not Revert_

### CODE REVIEW

File: backend/server/src/test/resources/migrations/testing/h2/1_release_2022_08.up.sql [hunk 1]
```diff
@@ -0,0 +1,1154 @@
...
+    "DEPLOYMENT_ID" CHARACTER VARYING(10)
+);
+-- 336 +/- SELECT COUNT(*) FROM DATABASE_CHANGE_LOG;
+INSERT INTO "DATABASE_CHANGE_LOG" VALUES('createTable-1', 'anandJ', 'liquibase/create-tables-changelog-1.0.xml', TIMESTAMP '2022-11-10 15:43:59.783151', 1, 'EXECUTED', '8:b64eccfe76381356d89452ebadacfdf7', 'createTable tableName=artifact_store; createTable tableName=comment; createTable tableName=dataset; createTable tableName=git_snapshot; createTable tableName=job; createTable tableName=path_dataset_version_info; createTable tableName=query_dataset...', '', NULL, '4.9.1', NULL, NULL, '8075239297');
+INSERT INTO "DATABASE_CHANGE_LOG" VALUES('create_artifact_store', 'anandJ', 'liquibase/create-tables-changelog-1.0.xml', TIMESTAMP '2022-11-10 15:43:59.801159', 2, 'MARK_RAN', '8:a1d122741f130094a332b6b93e3687e8', 'createTable tableName=artifact_store', '', NULL, '4.9.1', NULL, NULL, '8075239297');
+INSERT INTO "DATABASE_CHANGE_LOG" VALUES('create_comment', 'anandJ', 'liquibase/create-tables-changelog-1.0.xml', TIMESTAMP '2022-11-10 15:43:59.803158', 3, 'MARK_RAN', '8:b41aeb46a59fe79e4bc90d2ac24ba787', 'createTable tableName=comment', '', NULL, '4.9.1', NULL, NULL, '8075239297');
```
Comment (jkwatson-verta) [inline backend/server/src/test/resources/migrations/testing/h2/1_release_2022_08.up.sql:35]: whoa. I don't know that we need to be inserting anything into liquibase any more!

Comment (jkwatson-verta) [reply backend/server/src/test/resources/migrations/testing/h2/1_release_2022_08.up.sql:35]: especially for H2, where this data will definitely never be used.

**Review by jkwatson-verta — APPROVED** [review]
I don't think we need to be inserting stuff into the liquibase changelog table (or even creating it!), but let's get this merged and go from there.

File: backend/server/src/test/resources/migrations/testing/h2/1_release_2022_08.up.sql (hunk 1 above)
Comment (anandjakhaniya) [reply backend/server/src/test/resources/migrations/testing/h2/1_release_2022_08.up.sql:35]: Ok. I will remove it.

**Pull request merged**